static/app/components/group/suggestedOwners/suggestedOwners.tsx

```diff
@@ -24,48 +25,54 @@ type Props = {
   group: Group;
   event: Event;
   committers?: Committer[];
-};
+} & AsyncComponent['props'];
 
 type State = {
-  rules: Rules;
-  owners: Array<Actor>;
+  event_owners: {rules: Rules; owners: Array<Actor>};
   codeowners: CodeOwner[];
   isDismissed: boolean;
-};
-
-class SuggestedOwners extends React.Component<Props, State> {
-  state: State = {
-    rules: null,
-    owners: [],
-    codeowners: [],
-    isDismissed: true,
-  };
+} & AsyncComponent['state'];
+
+class SuggestedOwners extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      event: {rules: null, owners: []},
```

Comment:
I know this PR is refactoring in place but could we make `rules` an empty array?

Reply:
@mgaeta 👍  yea I think we should